manifests/client.pp

```diff
@@ -354,7 +354,7 @@
 
   file { "${etc_directory}/openvpn/${server}/download-configs/${name}/${name}.conf":
     owner   => root,
-    group   => $::openvpn::params::root_group,
+    group   => 0,
```

Comment:
Why did you switch away from the parameter to a hardcoded number? I don't think that there are a lot of usecases to set this to something else, but since the parameter is already there we could keep it?

Reply:
I would prefer to keep it, because the root group is always 0.

But its `backwards-incompatible`, expect you want this parameter.